src/components/EditTrip.js

```diff
@@ -0,0 +1,17 @@
+import { useHistory } from "react-router";
+
+export default function EditTrip() {
+  const history = useHistory();
+  return (
+    <div className="editTrip">
```

Comment:
Some CSS conventions info. I link here a comment i put to Rikki.
https://github.com/FriederikeM/routinely/pull/9#discussion_r620908613